std/stdio.d

```diff
@@ -3315,6 +3358,23 @@ unittest
 }
 
 /**
+Forwards to `stdout.print`.
+*/
+void print(string separator = " ", string eol = "\n", S...)(S args)
+{
+    trustedStdout.print!(separator, eol)(args);
+}
+
+///
+unittest
+{
+    void printCSVRow(Data...)(Data data)
+    {
+        print!", "(data);
```

Comment:
This wouldn't actually be legal csv.... but it might be if the print function could also take a per-item transformation function to quote it properly. csv technically shouldn't have a space after the comma either.

The transformation function might just be interesting though, but it might also complicate it more than is worth doing. Some kinda of dg(item.asString) that defaults to identity.